.pre-commit-config.yaml

```
@@ -15,6 +15,7 @@ repos:
     hooks:
       - id: clang-format
         types_or: [c++]
+        files: \.(c|cc|cxx|cpp|h|hpp|hxx|tpp)$
```

```suggestion
        files: \.(c|cpp|h|hpp|tpp)$
```
We only use these extensions right?